### ISSUE TITLE
Periodic Savon::UnknownOperationError when validating VAT

### ISSUE THREAD

**Issue by lvonk** [yolk/valvat#100]
Hi, we use `Valvat::Lookup.validate(vat)` (`1.1.0`) to validate VAT numbers. Approximately once per month for a short time period (<15 mins) we see a `Savon::UnknownOperationError` being raised with the following details:

```
Unable to find SOAP operation: :check_vat
Operations provided by your service: []
```

Could this be maintenance or something? I read in the docs then `nil` is returned, perhaps something changed on the VIES side?

**Comment by yolk** [comment]
This could be maintenance, but more likely the VIES-server is in an unstable state when returning this answer. Do you get this response for all kinds of VAT numbers, or only for numbers from specific states?

Any data to narrow down the issue would be helpful! Thanks!

**Comment by lvonk** [comment]
We are mostly targetting the dutch market, the last two occurrences were with VAT numbers from `NL` (Netherlands) and `BE` (belgium)

The last occurrences were:

<img width="179" alt="Screenshot 2021-03-22 at 22 20 45" src="https://user-images.githubusercontent.com/104943/112060167-1e35e500-8b5d-11eb-8801-adf4e8694dbf.png">
<img width="161" alt="Screenshot 2021-03-22 at 22 21 03" src="https://user-images.githubusercontent.com/104943/112060165-1d9d4e80-8b5d-11eb-8060-8156b6ce8fd8.png">

**Comment by aiomaster** [comment]
We've seen this exception at the same time for different countries (NL and AT, occurred at 17th May between 19h and 20h), so I think it is independend of the country.
Backtrace looks like:
```
/GEM_ROOT/gems/savon-2.12.1/lib/savon/operation.rb:23:in ensure_exists!
/GEM_ROOT/gems/savon-2.12.1/lib/savon/operation.rb:15:in create
/GEM_ROOT/gems/savon-2.12.1/lib/savon/client.rb:32:in operation
/GEM_ROOT/gems/savon-2.12.1/lib/savon/client.rb:36:in call
/GEM_ROOT/gems/valvat-1.1.0/lib/valvat/lookup/request.rb:18:in perform
/GEM_ROOT/gems/valvat-1.1.0/lib/valvat/lookup.rb:31:in response
/GEM_ROOT/gems/valvat-1.1.0/lib/valvat/lookup.rb:13:in validate
/GEM_ROOT/gems/valvat-1.1.0/lib/valvat/lookup.rb:20:in validate
```
If it is just another possibility how the VIES-service can be down, this gem should handle this case and throw some kind of `ViesError` maybe something like `ServiceUnavailable` instead.

**Comment by yolk** [comment]
Is this still a problem? v1.1.2 fixed a similar error.

**Comment by ArturT** [comment]
I was using v1.1.2 and I had a different error `Valvat::UnknownViesError`. I updated my [workaround code above](https://github.com/yolk/valvat/issues/100#issuecomment-947928782).

**Comment by yolk** [comment]
Fixed in v1.1.3
